common/core/src/commonMain/kotlin/com/trafi/ui/component/internal/ButtonConstants.kt

```diff
@@ -0,0 +1,16 @@
+package com.trafi.ui.component.internal
+
+import com.trafi.ui.theme.internal.BasicFontWeight.*
+import com.trafi.ui.theme.internal.ColorPalette.Gray200
+import com.trafi.ui.theme.internal.CurrentTheme
+import com.trafi.ui.theme.internal.copy
+
+class ButtonConstants(theme: CurrentTheme) {
+    val defaultColor = theme.colorPalette.primary
+    val defaultContentColor = theme.colorPalette.onPrimary
+    val disabledColor = Gray200
```

Comment:
Should't this color be different for dark theme?

Reply:
And it is not clear if this is disabled button background color or  disabled button content color

Reply:
Good questions 🙂 Just plain "color" refers to the background color, although we could definitely be more explicit if we want to.

Reply:
No, its ok to keep this naming convention 👍 Personally to me, disabled content color would be nice as white content color on a light gray background color doesn't seem to be nice(Specially when i'm not wearing my 👓) 😄 
<img width="372" alt="Screen Shot 2020-12-10 at 16 34 59" src="https://user-images.githubusercontent.com/14818488/101785628-a855c280-3b05-11eb-89f7-0101360b1165.png">